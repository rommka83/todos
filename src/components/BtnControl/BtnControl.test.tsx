import React from 'react';
import renderer from 'react-test-renderer';
import { BtnControl } from './index';

describe('проверка BtnControl', () => {
  test('отрисовка BtnControl', () => {
    const component = renderer.create(<BtnControl customValue='Текст кнопки' />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
