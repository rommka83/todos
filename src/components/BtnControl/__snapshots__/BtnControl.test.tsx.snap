// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`проверка BtnControl отрисовка BtnControl 1`] = `
<button
  className="root"
>
  Текст кнопки
</button>
`;
